gpii/node_modules/ontologyHandler/src/ontologyHandler.js

```diff
@@ -119,9 +142,9 @@ URLs - this aspect is taken care of in the preferences server component.
             fluid.log("ontologyHandler Loading filename " + filename);
             var transformSpec = fs.readFileSync(ontologyDir + "/" + filename, "utf8");
 
-            // remove ".json" part of filename to use as key
-            filename = filename.substring(0, filename.length - 5);
-            transformSpecs[filename] = JSON.parse(transformSpec);
+            // get the ontology name by removing the extension of the filename
+            var ontologyName = path.parse(filename).name;
```

Comment:
This variable name is misleading. It's not an ontology name at this point but a pair of them